apm/docs/metrics.md

```diff
@@ -1,610 +1,961 @@
-    <td>5分钟内每秒发送的交易请求总数</td>
-    <td>待补充</td>
-  </tr>
-  <tr>
-    <td>Success Total</td>
-    <td>sum(rate(muta_api_request_result_total{result="success",type="send_transaction"}[5m]))</td>
-    <td>5分钟内每秒发送的交易请求 Success 总数 </td>
-    <td>待补充</td>
-  </tr>
-  <tr>
-    <td>instance</td>
-    <td>rate(muta_api_request_result_total{result="success", type="send_transaction"}[5m])</td>
-    <td>5分钟内每个节点每秒发送交易请求 Success 数</td>
-    <td>待补充</td>
-  </tr>
-  <tr>
-    <td>current_height</td>
-    <td>instance</td>
-    <td>sort_desc(muta_consensus_height)</td>
-    <td>当前区块高度</td>
-    <td>待补充</td>
-  </tr>
-  <tr>
-    <td>synced_block</td>
-    <td>instance</td>
-    <td>muta_consensus_sync_block_total </td>
-    <td>每个节点同步区块总数</td>
-    <td>待补充</td>
-  </tr>
-  <tr>
-    <td>network_message_arrival_rate</td>
-    <td>value</td>
-    <td>((sum(muta_network_message_total{target="all", direction="sent"}) * 20) +<br> (sum(muta_network_message_total{target="single", direction="sent"})) )<br>/<br>sum(muta_network_message_total{direction="received"}) <br></td>
-    <td>5分钟内网络到达率</td>
-    <td>待补充</td>
-  </tr>
-  <tr>
-    <td>consensus_round_cost</td>
-    <td>instance</td>
-    <td>(muta_consensus_round &gt; 0 )</td>
-    <td>达成共识需要的轮次（常规为 0， view change 时，数值会发生改变）</td>
-    <td>待补充</td>
-  </tr>
-  <tr>
-    <td>mempool_cached_tx</td>
-    <td>instance</td>
-    <td>muta_mempool_tx_count</td>
-    <td>当期交易池中的交易数量</td>
-    <td>待补充</td>
-  </tr>
-</tbody>
-</table>
\ No newline at end of file
+#### TPS
+- description: TPS for consensus
+<details>
+<summary>Legende details</summary>
+
+##### TPS
+TPS for consensus
+```
+avg(rate(muta_consensus_committed_tx_total[5m]))
+```
+</details>
+
+
+#### consensus_p90
+- description: Consensus time for P90
+<details>
+<summary>Legende details</summary>
+
+##### time_usage(s)
+Consensus time for P90
+```
+avg(histogram_quantile(0.90, sum(rate(muta_consensus_duration_seconds_bucket[5m])) by (le, instance)))
+```
+</details>
+
+#### exec_p90
+- description: Consensus exec time for P90
+<details>
+<summary>Legende details</summary>
+
+##### /
+Consensus exec time for P90
+```
+avg(histogram_quantile(0.90, sum(rate(muta_consensus_time_cost_seconds_bucket{type="exec"}[5m])) by (le, instance)))
+```
+</details>
+
+#### put_cf_each_block_time_usage
+- description: Average time to write a block height
+<details>
+<summary>Legende details</summary>
+
+##### /
+Average time to write a block height
+```
+avg (sum by (instance) (increase(muta_storage_put_cf_seconds[5m]))) / avg(increase(muta_consensus_height[5m]))
+```
+</details>
+
+#### get_cf_each_block_time_usage
+- description: Average time to read a block height
+<details>
+<summary>Legende details</summary>
+
+##### /
+Average time to read a block height
+```
+avg (sum by (instance) (increase(muta_storage_get_cf_seconds[5m]))) / avg(increase(muta_consensus_height[5m]))
+```
+</details>
+
+
+#### processed_tx_request
+- description: Monitor transaction requests
```

Comment:
received transaction request count in last 5 minutes (the unit is _count/second_)